internal/provider/compliance_framework_data_source.go

```diff
@@ -0,0 +1,248 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	mondoov1 "go.mondoo.com/mondoo-go"
+)
+
+var _ datasource.DataSource = (*complianceFrameworkDataSource)(nil)
+
+func NewComplianceFrameworkDataSource() datasource.DataSource {
+	return &complianceFrameworkDataSource{}
+}
+
+type complianceFrameworkDataSource struct {
+	client *ExtendedGqlClient
+}
+
+type complianceFrameworkDataSourceModel struct {
+	SpaceID              types.String               `tfsdk:"space_id"`
+	SpaceMrn             types.String               `tfsdk:"space_mrn"`
+	ComplianceFrameworks []complianceFrameworkModel `tfsdk:"compliance_frameworks"`
+}
```

Comment:
@chris-rock if we want to rename it to `mondoo_frameworks`, then we should rename the whole resource to not contain `compliance` in its  name. This way it would be uniform.

```suggestion
func NewFrameworksDataSource() datasource.DataSource {
	return &frameworksDataSource{}
}

type frameworksDataSource struct {
	client *ExtendedGqlClient
}

type frameworksDataSourceModel struct {
	SpaceID              types.String               `tfsdk:"space_id"`
	SpaceMrn             types.String               `tfsdk:"space_mrn"`
	Frameworks []frameworksModel `tfsdk:"frameworks"`
}
```

Reply:
Additionally should we the remove the `compliance_` part from all our resources?

E.g. `compliance_framework_resource.go` --> `framework_resource.go`  or `custom_compliance_framework_resource.go` --> `custom_framework_resource.go`

In this way all resources would have uniform naming shema.

Reply:
And should we also rename:
```
data "mondoo_policy" "policy" {
  space_id      = "your-space-1234567"
  catalog_type  = "ALL" # available options: "ALL", "POLICY", "QUERYPACK"
  assigned_only = true
}
```

to 

```
data "mondoo_policies" "policy" {
  space_id      = "your-space-1234567"
  catalog_type  = "ALL" # available options: "ALL", "POLICY", "QUERYPACK"
  assigned_only = true
}
```

Reply:
@mati007thm I agree with all your naming suggestions. Lets do it.